src/package.xml

```diff
@@ -794,6 +807,7 @@
         <members>Account-HEDA Organization Layout</members>
         <members>Address__c-HEDA Address Layout</members>
         <members>Affiliation__c-HEDA Affiliation Layout</members>
+        <members>Attribute__c-HEDA Attribute Layout</members>
```

Comment:
@xinnanlisf This might apply to the other open PR too: I think we said that all new layouts added after the EDA name change will use "EDA" now instead of "HEDA" (I realize this was started prior to the name change)

Reply:
yea since it's not merged yet and packaged we should change this. good catch.